04_documentation/markup_languages_slides.md

```diff
@@ -0,0 +1,254 @@
+---
+type: slide
+slideOptions:
+  transition: slide
+  width: 1400
+  height: 900
+  margin: 0.1
+---
+
+<style>
+  .reveal strong {
+    font-weight: bold;
+    color: orange;
+  }
+  .reveal p {
+    text-align: left;
+  }
+  .reveal section h1 {
+    color: orange;
+  }
+  .reveal section h2 {
+    color: orange;
+  }
+  .reveal code {
+    font-family: 'Source Code Pro';
+    color: orange;
+  }
+  .reveal section img {
+    background:none;
+    border:none;
+    box-shadow:none;
+  }
+</style>
+
+# Lightweight Markup Languages
+
+---
+
+## Learning Goals
+
+- You know common markup languages.
+- You know important parts of the syntax of these languages.
+- You know what benefits these languages have.
+- You know how to convert files written in these languages.
+
+---
+
+## Do you know any (Lightweight) Markup Languages?
+
+---
+
+## Introduction
+
+- Why "lightweight"/"simple"/"humane"?
+    - Readable without special rendering -> readable in text editor
+    - Originates from earlier times when texts on screens would not support special formatting
+- Different languages with different goals
+    - Presentation: `Markdown`, `reStructuredText`, `AsciiDoc`, [WhatsApp](https://faq.whatsapp.com/general/chats/how-to-format-your-messages/?lang=en)...
+    - Data serialization/storage: `YAML`, `JSON`...
+
+See [Wikipedia](https://en.wikipedia.org/wiki/Lightweight_markup_language)
+
+---
+
+## Common Properties
+
+- **Whitespaces** very important
+- Suitable for **conversion** into other formats, especially HTML
+- Plain text files -> Good for us
+
+  > "If you can't git diff a file format, it's broken."
+
+    - Suitable for [Docs as code](https://www.writethedocs.org/guide/docs-as-code/) and [DocOps](https://www.writethedocs.org/guide/doc-ops/)
+
+---
+
+## Docs as Code
+
+> Documentation as Code (Docs as Code) refers to a philosophy that you should be writing documentation with the same tools as code
+>
+> - Issue Trackers
+> - Version Control (Git)
+> - Plain Text Markup (Markdown, reStructuredText, Asciidoc)
+> - Code Reviews
+> - Automated Tests
+>
+> [Write the Docs](https://www.writethedocs.org/guide/docs-as-code/)
```

Comment:
In my slides, I did not put such complete things in a comment, even when I copied 1:1 from a source. I think it's OK to just say "From [Write the Docs](...)". Where should we stop with this style otherwise?

Reply:
I removed the list now. I am not sure what you mean with "this style". On the slides I went to take the first sentence of the corresponting homepage.